apis/duck/v1alpha1/retired_targetable_types.go

```diff
@@ -24,11 +24,15 @@ import (
 	"github.com/knative/pkg/apis/duck"
 )
 
-// Targetable is very similar concept as Sinkable. However, at the
-// transport level they have different contracts and hence Sinkable
-// and Targetable are two distinct resources.
-
-// Targetable is the schema for the targetable portion of the payload
+// Targetable is an earlier version of the Callable interface.
+// Callable is a higher-level interface which implements Addressable
+// but further promises that the destination may synchronously return
+// response messages in reply to a message.
+//
+// Targetable implementations should instead implement Addressable and
+// include an `eventing.knative.dev/returns=any` annotation.
+
+// Targetable is retired; implement Addressable for now.
```

Comment:
The language in other parts of the repo is `// Deprecated: Use XYZ` but perhaps this is not deprecated as much as it is deleted?

Reply:
I think in Eventing repo so many things have not been deprecated but outright changed, since these support them, how about we just delete things instead of leave cruft here.

Reply:
I was thinking that I'd update deps in serving and then switch the types, but I think I can just do the `dep` and caller update in one pass.